From 74e3e1181320b5154e40af12c0db83e039412377 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Sat, 16 Mar 2019 12:28:11 +0800
Subject: [PATCH 6/7] Revert "sepolicy: Allow perf HAL to set freq props"

 * ro.min_freq_0 and ro.min_freq_4 are renamed to ones with vendor.
   prefix, and labeled as vendor_mpctl_prop

This reverts commit 4a25e1746040ddb3a79c7620df1736d00e63f146.

Change-Id: Iab76825809122e56ea7cb8473f85816750f7b218
---
 vendor/common/hal_perf_default.te | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/vendor/common/hal_perf_default.te b/vendor/common/hal_perf_default.te
index eb0fbff6..4083965c 100644
--- a/vendor/common/hal_perf_default.te
+++ b/vendor/common/hal_perf_default.te
@@ -41,6 +41,7 @@ allow hal_perf_client hal_perf_hwservice:hwservice_manager find;
 allow hal_perf cgroup:file r_file_perms;
 allow hal_perf_default proc:file rw_file_perms;
 allow hal_perf device_latency:chr_file rw_file_perms;
+allow hal_perf freq_prop:file r_file_perms;
 allow hal_perf vendor_mpctl_prop:file r_file_perms;
 allow hal_perf_default mpctl_data_file:dir rw_dir_perms;
 allow hal_perf_default mpctl_data_file:file create_file_perms;
@@ -51,9 +52,6 @@ hal_client_domain(hal_perf_default, hal_graphics_composer);
 allow hal_perf_default qdisplay_service:service_manager find;
 r_dir_file(hal_perf_default, appdomain);
 
-# Allow perf HAL to set freq props
-set_prop(hal_perf_default, freq_prop)
-
 allow hal_perf {
     sysfs_devices_system_cpu
     sysfs_mpdecision
-- 
2.17.1

